src/utils/actions.ts

```diff
@@ -9,7 +9,7 @@ export type ActionMeta = {
  * stopForwarding allows you to give it an action, and it will return an
  * equivalent action that will only play in the current process
  */
-export const stopForwarding = (action: FluxStandardAction<ActionMeta>) => ({
+export const stopForwarding = (action: FluxStandardAction<ActionMeta>): any => ({
```

Comment:
Why explicit `any`?

Reply:
I was having a hard time trying to make the store typings comply, the was the only one which I could't find a solution for.

Reply:
Is this the last obstacle to merging this PR?